lms/static/sass/features/_course-upgrade-message.scss

```diff
@@ -0,0 +1,118 @@
+/*
+  NOTE: If you make significant changes to the design, remember to update the Segment event properties and change
+  the creative property. This will allow us to better track individual performance of each style of the message.
+  Search for the courseware_verified_certificate_upsell promotion ID.
+ */
+$upgrade-message-background-color: #005686;
+$upgrade-message-button-color: #008100;
+
+// Expanded upgrade message
+.vc-message {
+  background: $upgrade-message-background-color;
+  color: white;
+  padding: 20px;
+  position: relative;
+  margin: 0 0 20px;
+  // CSS animation for smooth height transition
+  -webkit-transition: all 0.2s ease-in-out;
+  transition: all 0.2s ease-in-out;
+  &:after {
+    content: "";
+    display: table;
+    clear: both;
+  }
+  // Message copy
+  h3 {
+    font-size: 1.25rem;
+    font-weight: 700;
+    margin-bottom: 1rem;
+    float: left;
+  }
+  ul {
+    clear: left;
+    padding-left: 0;
+    font-size: 0.825rem;
+    margin: 1rem 0 1rem;
+    display: table;
+    > li {
+      list-style: none;
+      display: table-row;
+      &:before {
+        content: "\2022";
+        display: table-cell;
+        padding-right: 10px;
+      }
+      &:after {
+        content: "";
+        display: table-row;
+        height: 0.25rem;
+      }
+    }
+  }
+  img {
+    max-width: 100%;
+  }
+  // Show/hide button
+  .vc-toggle {
+    float: right;
+    color: white;
```

Comment:
Use a color constant here so that themes can control it, or make it transparent.